maraboupy/Marabou.py

```diff
@@ -120,7 +120,8 @@ def solve_query(ipq, filename="", verbose=True, options=None):
 def createOptions(numWorkers=1, initialTimeout=5, initialDivides=0, onlineDivides=2,
                   timeoutInSeconds=0, timeoutFactor=1.5, verbosity=2, snc=False,
                   splittingStrategy="auto", sncSplittingStrategy="auto",
-                  restoreTreeStates=False, splitThreshold=20, solveWithMILP=False ):
+                  restoreTreeStates=False, splitThreshold=20, solveWithMILP=False,
+                  preprocessorBoundTolerance=0.0000000001):
```

Comment:
This is the same initialization value as in Options.cpp, right? Why do we need to initialize in two places? If we leave it uninitialized here, won't it default to the value in Options.cpp?

Reply:
This value must be initialized with any value, since it is the (not the first and actually the) last parameter in a list of initialized function's parameters. Otherwise, the interpreter raise an exception.

In addition, if I move it to be the first parameter in the function it can make damage to every call without this parameter. It will not backward compatible, and also won't use the default value from the cpp file. It is possible, but in my opinion, the current option is better.

Maybe it is possible to take the default value using some python-cpp API, but it seems that default values can't be supported (at least while using pybind, since they are not part of the signature of the function in cpp). Anyway, the whole function uses default variables which are defined in the cpp file too, so i think it should be solved in another PR.